src/Tribe/Log/Service_Provider.php

```diff
@@ -145,4 +147,30 @@ public function add_logging_engine( array $logging_engines = [] ) {
 
 		return $logging_engines;
 	}
+
+	/**
+	 * Enables logging in the service provider, if not already enabled.
+	 *
+	 * @since TBD
+	 */
+	public function enable(  ) {
+		if ( has_action( 'tribe_log', [ $this, 'dispatch_log' ] ) ) {
+			return;
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 802ff1e85050027188bdce92bcf694c6</sub>